kubectl-rook-ceph.sh

```diff
@@ -226,14 +227,202 @@ function path_cm_rook_ceph_operator_config() {
+    --mon-initial-members=$ROOK_CEPH_MON_INITIAL_MEMBERS \
+    --id=$good_mon \
+    --foreground \
+    --public-addr=$good_mon_public_ip \
+    --setuser-match-path=/var/lib/ceph/mon/ceph-$good_mon/store.db \
+    --public-bind-addr=$ROOK_POD_IP \
+    --extract-monmap=$monmap_path
+
+  echo -e "\nPrinting monmap"; \
+  KUBECTL_NS_OPERATOR exec deploy/rook-ceph-mon-$good_mon-debug -c mon -- monmaptool --print $monmap_path
+
+  # remove all the mons except the good one
+  for bad_mon in "${bad_mons[@]}"
+  do
+    echo -e "\nRemoving mon $bad_mon"
+    KUBECTL_NS_OPERATOR exec deploy/rook-ceph-mon-$good_mon-debug -c mon -- monmaptool $monmap_path --rm $bad_mon
+  done
+
+  echo -e "\rInjecting the monmap"
+  KUBECTL_NS_OPERATOR exec deploy/rook-ceph-mon-$good_mon-debug -c mon -- ceph-mon \
+    --fsid=$ceph_fsid \
+    --keyring=/etc/ceph/keyring-store/keyring \
+    --log-to-stderr=true \
+    --err-to-stderr=true \
+    --mon-cluster-log-to-stderr=true \
+    --log-stderr-prefix=debug \
+    --default-log-to-file=false \
+    --default-mon-cluster-log-to-file=false \
+    --mon-host=$ROOK_CEPH_MON_HOST \
+    --mon-initial-members=$ROOK_CEPH_MON_INITIAL_MEMBERS \
+    --id=$good_mon \
+    --foreground \
+    --public-addr=$good_mon_public_ip \
+    --setuser-match-path=/var/lib/ceph/mon/ceph-$good_mon/store.db \
+    --public-bind-addr=$ROOK_POD_IP \
+    --inject-monmap=$monmap_path
+  echo -e "\nFinished updating the monmap!"
+  set -eu
+
+  echo -e "\nPrinting final monmap"
+  KUBECTL_NS_OPERATOR exec deploy/rook-ceph-mon-$good_mon-debug -c mon -- monmaptool --print $monmap_path
+
+  echo -e "\nReseting the mons in the rook-ceph-mon-endpoints configmap to the good mon"
+  KUBECTL_NS_OPERATOR patch configmaps rook-ceph-mon-endpoints --type json --patch "[{ op: replace, path: /data/data, value: $good_mon=$good_mon_public_ip:$good_mon_port }]"
+
+  echo -e "\nStopping the debug pod for mon $good_mon"
+  run_stop_debug rook-ceph-mon-$good_mon
+
+  echo -e "\nCheck that the good mon is responding"
+  wait_for_mon_status_response
+
+  echo -e "\nMon quorum was successfully reset to mon $good_mon"
+
+  echo -e "\nPurging the bad mons: ${bad_mons[*]}"
+  for bad_mon in "${bad_mons[@]}"
+  do
+    echo "purging old mon: $bad_mon"
+    KUBECTL_NS_OPERATOR delete deploy rook-ceph-mon-$bad_mon
+    KUBECTL_NS_OPERATOR delete svc rook-ceph-mon-$bad_mon
+  done
+
+  echo -e "\nStart up the operator and expand to full mon quorum again? If so, type \"yes\""
+  read START_OPERATOR
+  if [ "$START_OPERATOR" = "yes" ]; then
+    # scale up the operator
+    KUBECTL_NS_CLUSTER scale deployment rook-ceph-operator --replicas=1
+  fi
+}
+
+function wait_for_mon_status_response() {
+  i=0
+  max_retries=20
+  sleep_time=5
+
+  exit_status=1
+  while [[ $exit_status != 0 ]]
+  do
+    # Don't fail the script if the ceph command fails
+    set +e
+    KUBECTL_NS_OPERATOR exec deploy/rook-ceph-tools -- ceph status --connect-timeout=3
```

Comment:
```suggestion
    KUBECTL_NS_CLUSTER exec deploy/rook-ceph-tools -- ceph status --connect-timeout=3
```
also we have method to print ceph command can we use that?